docs/ref/forms/fields.txt

```diff
@@ -275,6 +275,35 @@ fields. We've specified ``auto_id=False`` to simplify the output:
     <div>Sender:<div class="helptext">A valid email address, please.</div><input type="email" name="sender" required></div>
     <div>Cc myself:<input type="checkbox" name="cc_myself"></div>
 
+When a field has help text and :attr:`~django.forms.BoundField.id_for_label`
+returns a value, we associate``help_text`` with the input using
+``aria-describedby``:
+
+    >>> from django import forms
+    >>> class UserForm(forms.Form):
+    ...     username = forms.CharField(max_length=10, help_text='e.g., user@example.com')
+    >>> f = UserForm()
+    >>> print(f.as_div())
+    <div><label for="id_username">Username:</label>
+    <div class="helptext" id="id_username_helptext">e.g., user@example.com</div>
+    <input type="text" name="username" maxlength="10" required aria-describedby="id_username_helptext" id="id_username">
+    </div>
+    
+When adding a custom ``aria-describedby`` attribute, make sure to also include the id of the ``help_text`` element (if used).
+
+    >>> class UserForm(forms.Form):
+    ...     username = forms.CharField(
+    ...         max_length=10, help_text='e.g., user@example.com',
+    ...         widget=forms.TextInput(attrs={'aria-describedby': 'custom-description'}),
+    ...     )
+    >>> f = UserForm()
+    >>> print(f['username'])
+    <input type="text" name="username" aria-describedby="custom-description" maxlength="10" id="id_username" required>
```

Comment:
Shouldn’t this be adapted to show _correct_ usage of manually adding the id of `help_text`?

```suggestion
    >>> class UserForm(forms.Form):
    ...     username = forms.CharField(
    ...         max_length=10, help_text='e.g., user@example.com',
    ...         widget=forms.TextInput(attrs={'aria-describedby': 'id_username_helptext custom-description'}),
    ...     )
    >>> f = UserForm()
    >>> print(f['username'])
    <input type="text" name="username" aria-describedby="id_username_helptext custom-description" maxlength="10" id="id_username" required>
```

Reply:
Fixed.